MAUI/ImageEditor/Effects.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: Image Effects in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about Image Effect support in Syncfusion .NET MAUI Image Editor (SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+# Image Effect in .NET MAUI Image Editor (SfImageEditor)
+
+With the image editor control, you can apply various effects such as Brightness, Hue, Saturation, Contrast, Blur, Opacity, and Sharpen to your image. These effects can be applied using the toolbar or by utilizing the ImageEffect method. The ImageEffect method consists of two arguments: ImageEffect and EffectValue. The ImageEffect is an enumeration that includes the following effects:
+
+* Brightness
+* Blur
+* Contrast
+* Exposure
+* Hue
+* Saturation
+* Sharpen 
+* Opacity
+* None
+
+The EffectValue are the corresponding ImageEffect values, which varies for each effect, and they are explained as follows.
+
+N> The Image Effect enum also contains “None” option, which removes all the previously applied effects, which are not saved.
+The ImageEffect method only applies the effect to preview image, if you want to save the applied effect you should call SaveEdits method.
+
+## Brightness
+
+Brightness is used to adjust the overall lightness or darkness of the image. The value of the brightness effect ranges from -1 to 1 and the default value is 0.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect.Brightness, -0.6);
+    . . .
+}
+
+{% endhighlight %}
+
+
+## Blur
+
+Blur is used to Creates a soft and unfocused appearance by reducing the image's sharpness. The value of blur effect ranges from 0 to 1 and the default value is 0.

```

Comment:
Blur is used to create a soft and unfocused appearance by reducing the image's sharpness. The value of blur effect ranges from 0 to 1 and the default value is 0.

Reply:
content updated